### DISCUSSION TITLE
Text colors, imports not connecting?

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#83] [Q&A]
Ok, so it may very well be true that I ask a lot of silly questions.  Here I'm wondering why my `from brownie import Lottery, accounts, config, network` text colors on **brownie** and **network** are different that yours.  I assume it means something hasn't properly activated somewhere.

## Mine
![image](https://user-images.githubusercontent.com/26919212/134544862-8749cb55-8017-4103-aa5e-48e1f39f841b.png)

## Yours
![image](https://user-images.githubusercontent.com/26919212/134544999-17c4e542-c214-4359-8134-4ba6c9ba6c3f.png)

**Comment by toddabraham** [comment]
UPDATE: brownie is now connected after a VSCode restart.  Network however, continues to seem not to be connected, but not seemingly preventing tests from running.

![image](https://user-images.githubusercontent.com/26919212/134636996-3b3307b3-3cc0-42ff-89b1-1987d189f2b2.png)

![image](https://user-images.githubusercontent.com/26919212/134637087-cfd2e6a9-9889-4d81-9155-12083ed2a1d1.png)

`exceptions` also not connecting

![image](https://user-images.githubusercontent.com/26919212/134643776-0d669f3f-fd1a-474f-b20c-eb22716fc971.png)

**Reply by AlbertoSinigaglia** [reply]
this happens also in PyCharm that is way more integrated with the Python env... so I think there is nothing to do to get those working, I've tried pretty much everything I can find on StackOverflow

**Reply by PatrickAlphaC** [reply]
Yes....
